assets/js/googlesitekit/api/index.js

```diff
@@ -21,21 +21,21 @@ let cachingEnabled = true;
 /**
  * Make a request to a WP REST API Site Kit endpoint.
  *
- * @param {string} type        The data to access. One of 'core' or 'modules'.
- * @param {string} identifier  The data identifier, eg. a module slug like `'search-console'`.
- * @param {string} datapoint   The endpoint to request data from.
- * @param {Object} queryParams Query params to send with the request.
- * @param {Object} data        Request data to send.
- * @param {Object} useCache    Set to `true` to use the caching. Caching is only used for `GET` requests.
+ * @param {string}  type                The data to access. One of 'core' or 'modules'.
+ * @param {string}  identifier          The data identifier, eg. a module slug like `'search-console'`.
+ * @param {string}  datapoint           The endpoint to request data from.
+ * @param {Object}  options             Options to pass to the request
+ * @param {number}  options.cacheTTL    The oldest cache data to use, in seconds.
+ * @param {Object}  options.data        Request data to send.
+ * @param {number}  options.method      HTTP method to use for this request.
+ * @param {Object}  options.queryParams Query params to send with the request.
+ * @param {boolean} options.useCache    Set to `true` to use the caching. Caching is only used for `GET` requests.
```

Comment:
I think this should also say `Enable or disable caching for this request only.` - if the current default is true for example, you don't need to pass true.